pages/configuration/https.rst

```diff
@@ -196,7 +209,7 @@ The resulting PKCS#8 private key (``graylog-key.pem``) and the X.509 certificate
 Sample files
 ============
 
-This section show the difference between following private key formats with samples.
+This section show the difference between following private key formats with samples. It will help you to identify what file you have.
```

Comment:
typo - should be "This section shows..."  

ALSO, try "...between the following private key formats and provides samples.